scripts/install.sh

```diff
@@ -1,49 +1,98 @@
 #!/bin/bash
-set -e
+set -euo pipefail
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
 
-# Dafault inputs.
-KEEP_ETHEREUM_PASSWORD_DEFAULT="password"
 KEEP_CORE_PATH=$PWD
+KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity"
+
+# Defaults, can be overwritten by env variables/input parameters
 CONFIG_DIR_PATH_DEFAULT="$KEEP_CORE_PATH/configs"
+NETWORK_DEFAULT="local"
+KEEP_ETHEREUM_PASSWORD=${KEEP_ETH_ACCOUNT_PASSWORD:-"password"}
+CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY:-""}
 
-# Read user inputs.
-read -p "Enter ethereum accounts password [$KEEP_ETHEREUM_PASSWORD_DEFAULT]: " ethereum_password
-KEEP_ETHEREUM_PASSWORD=${ethereum_password:-$KEEP_ETHEREUM_PASSWORD_DEFAULT}
+help()
+{
+   echo -e "\nUsage: ENV_VAR(S) $0"\
+           "--config-dir <path>"\
+           "--network <network>"
+   echo -e "\nEnvironment variables:\n"
+   echo -e "\tKEEP_ETH_ACCOUNT_PASSWORD: Unlock an account with a password. Default password is 'password'"
+   echo -e "\tCONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY: Contracts owner private key on Ethereum"
+   echo -e "\nCommand line arguments:\n"
+   echo -e "\t--config-dir: Path to keep-core client configuration file(s)"
+   echo -e "\t--network: Host chain network for keep-core client."\
```

Comment:
Maybe `Ethereum network for...`? One should not use alfajores network with this script.

Reply:
Right, but I guess we can interpret a host chain as Ethereum local, Ethereum Ropsten or Mainnet, not only Ethereum vs Celo. Anyway, changed in b15cc9c